testing/tests/DevExpress.viz.core.series/areaSeries.tests.js

```diff
@@ -1068,17 +1163,56 @@ function setDiscreteType(series) {
             'hover': {
                 'fill': 'h-color',
                 'hatching': 'h-hatching',
-                opacity: undefined
+                opacity: undefined,
+                'filter': true,
             },
             'normal': {
                 'fill': 'n-color',
                 opacity: 'opacity',
-                hatching: undefined
+                hatching: undefined,
+                'filter': undefined,
             },
             'selection': {
                 'fill': 's-color',
                 'hatching': 's-hatching',
-                opacity: undefined
+                opacity: undefined,
+                'filter': true,
+            }
+        });
+    });
+
+    QUnit.test('custom styles are defined', function(assert) {
```

Comment:
really common. Maybe like this? Please, review all tests. 
Test name should describe the full case and answer the question "how should it work"?
Maybe like this?
```suggestion
    QUnit.test('series custom hover and selection styles should be passed to legend styles', function(assert) {
```

Reply:
The module has already name that all tests about legend styles. And the tests in this module check not only hover and selection. It check all styles: normal, hover and selection. Should it has me more ditailed describing?

Reply:
About legend series - got it. I'm just not used to old modules when tests are not inside of a module physically, so didn't catch it during review.

About detailed describing - yes. Now when u're trying to imagine a name for a test which tests hover, selection and normal styles, u can see another problem - this test is not atomic, it checks a lot of things at one time.